c++/include/celerite2/forward.hpp

```diff
@@ -297,7 +297,7 @@ void general_matmul_lower(const Eigen::MatrixBase<Input> &t1,               // (
 
   typedef typename LowRank::Scalar Scalar;
   typedef typename Eigen::internal::plain_col_type<Coeffs>::type CoeffVector;
-  typedef typename Eigen::Matrix<Scalar, LowRank::ColsAtCompileTime, RightHandSide::ColsAtCompileTime> Inner;
+  typedef typename Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor> Inner;
```

Comment:
From my experience, this change will dramatically impact performance because Eigen won't be able to generate properly vectorized code for small systems. It's really useful to compile for specific sizes! Why did you make this change?